docs/api/annotations.md

```diff
@@ -0,0 +1,233 @@
+# Annotations
+
+An annotation assigns pronunciation errors to specific fragments of an audio recording performed the
+an annotator user.
```

Comment:
I guess something went wrong with this sentence. Proposal:

Annotations are notes to specific fragments of audio. These annotations can then be classified. Common classifiers are `Prompt`, `Out of Prompt` or `Error`. Annotations are made by users called annotators. They are able to put these annotations on recorded audio.